java-frontend/src/test/java/org/sonar/java/se/NullableAnnotationUtilsTest.java

```diff
@@ -21,26 +21,30 @@
 
 import java.lang.reflect.Constructor;
 import java.lang.reflect.Modifier;
+import java.util.List;
 import java.util.stream.Stream;
-import org.junit.BeforeClass;
+import org.junit.Before;
 import org.junit.Test;
 import org.sonar.java.resolve.SemanticModel;
 import org.sonar.plugins.java.api.semantic.Symbol;
+import org.sonar.plugins.java.api.semantic.SymbolMetadata;
 import org.sonar.plugins.java.api.semantic.Type;
 import org.sonar.plugins.java.api.tree.MethodTree;
+import org.sonar.plugins.java.api.tree.NewArrayTree;
 
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.sonar.java.se.NullableAnnotationUtils.isAnnotatedNonNull;
 import static org.sonar.java.se.NullableAnnotationUtils.isAnnotatedNullable;
 import static org.sonar.java.se.NullableAnnotationUtils.isAnnotatedWith;
 import static org.sonar.java.se.NullableAnnotationUtils.isGloballyAnnotatedWith;
+import static org.sonar.java.se.NullableAnnotationUtils.valuesForGlobalAnnotation;
 
 public class NullableAnnotationUtilsTest {
 
-  private static SemanticModel semanticModel;
+  private SemanticModel semanticModel;
 
-  @BeforeClass
-  public static void setUp() {
+  @Before
+  public void setUp() {
```

Comment:
This change implies parsing and semantic computation for every test of this class. This seems quite some overhead.